packages/modal-ui-js/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "@near-wallet-selector/modal-ui-js",
+  "version": "0.0.1",
+  "type": "commonjs"
```

Comment:
Can you make update the version and make it to the current one `6.0.1`?
Also, you will need to remove the `type` property.